source/strings.tex

```diff
@@ -3010,7 +3010,7 @@
 A directive that informs a \tcode{basic_string} of a planned change in size,
 so that the storage allocation can be managed accordingly.
 After
-\tcode{reserve()},
+\tcode{reserve},
```

Comment:
Could we use slightly more regular wording here, something like "after a call of `reserve`" (or "an invocation of")?

Reply:
Updated to say "After a call to `reserve`". I hope it's clear that means _immediately_ after, not for the rest of the program!

Reply:
Then why not just say "immediately after"? Hope is not a strategy!

Reply:
Would "Following a call to `reserve`" be better?

Reply:
Works for me!

Reply:
"Following an invocation of `reserve`", does that sound more contemporary?

Reply:
@jwakely Did you want to do any of those ^^^ ?

Reply:
I think "Following a call to `reserve`" is my preference, updated.